install/index.markdown

```diff
@@ -45,11 +38,11 @@ _Note for Ubuntu 13.4 32 bit users_: There is a bug with GCC 4.7 on Ubuntu 13.4
     <div id="collapseTwo" class="accordion-body collapse">
       <div class="accordion-inner">
         <div class="toggle_content" style="display: block;"><p></p>
-          <h4>STEP 1: Ubuntu Installation: Debian Packages for MoveIt!</h4>
+          <h4>Step 1: Ubuntu Installation: Debian Packages for MoveIt!</h4>
           <div><div id="highlighter_390226" class="syntaxhighlighter nogutter  bash"><table border="0" cellpadding="0" cellspacing="0"><tbody><tr><td class="code"><div class="container"><div class="line number1 index0 alt2 highlighted"><code class="bash functions">sudo apt-get install ros-indigo-moveit-full</code></div></div></td></tr></tbody></table></div></div>
-          <h4>STEP 2: Debian Packages for MoveIt! with the PR2</h4>
+          <h4>Step 2: Debian Packages for MoveIt! with the PR2</h4>
           <p>These are not yet fully available for the PR2.</p>
```

Comment:
There's no reason not to have it here, as moveit_full_pr2 contains many of the tutorials.

Especially as you just halved the size of the page, I think it should be fine without expanding structures, yes.